Command/FixturesLoadCommand.php

```diff
@@ -178,7 +178,7 @@ protected function loadFixtures(InputInterface $input, OutputInterface $output,
             return;
         }
 
-        $nb = $loader->load($datas, $connectionName);
+        $nb = $loader->load($connectionName, $data);
```

Comment:
If we change the spelling to `$data` here, we need to change it above, too ;)

Reply:
we don't, that's a typo